client/client.js

```diff
@@ -0,0 +1,169 @@
+var assert = require('assert');
+var _ = require('lodash');
+var boot = require('loopback-boot');
+var debug = require('debug')('MeshModels.client');
+var loopback = require('loopback');
+var request = require('request');
+var url = require('url');
+var util = require('util');
+
+function Client(apiUrl) {
+  // Normalize the URI
+  var endpoint = url.parse(apiUrl);
+  debug('normalize endpoint %j', endpoint);
+  endpoint.pathname = '/api'; // Loopback is mounted here
+  endpoint.hostname = endpoint.hostname || 'localhost'; // Allow `http://:8888`
+  delete endpoint.host; // So .hostname and .port are used to construct URL
+  this.apiUrl = url.format(endpoint);
+
+  debug('connecting to %s', this.apiUrl);
+
+  var client = loopback();
+  client.dataSource('remote', {'connector': 'remote', 'url': this.apiUrl});
+  boot(client, __dirname);
+
+  this.loopback = client;
+  this.models = client.models;
+
+  // Resolve the models
+  client.models();
```

Comment:
this is a loopback issue .. `client.models()` populates the cache of models so you can access them with `client.models.ModelName`